test/test_helper.rb

```diff
@@ -0,0 +1,13 @@
+ENV["RAILS_ENV"] ||= "test"
+require_relative "../config/environment"
+require "rails/test_help"
+
+class ActiveSupport::TestCase
```

Comment:
if you are planning to use just mini test, then add the capybara settings here, and at least one system test that opens homepage

Reply:
Thank you I'm going to use RSpec, in the next PRs after integrating Solidus I'll configure FactoryBot and a Driver